demo/api/src/app.module.ts

```diff
@@ -42,6 +47,30 @@ import { PredefinedPageModule } from "./predefined-page/predefined-page.module";
 import { ProductsModule } from "./products/products.module";
 import { RedirectScope } from "./redirects/dto/redirect-scope";
 
+const createRemoteSchema = async (url: string): Promise<SubschemaConfig> => {
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+    const executor = async ({ document, variables }: ExecutionRequest) => {
+        const query = print(document);
+        const fetchResult = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({ query, variables }),
+        });
+        return fetchResult.json();
+    };
+    return {
+        schema: await schemaFromExecutor(executor),
+        executor: executor,
+        transforms: [
+            new RenameTypes((name) => `Foo${name}`),
+            new RenameRootTypes((name) => `Foo${name}`),
+            new RenameRootFields((operationName, fieldName, fieldConfig) => `foo_${fieldName}`),
```

Comment:
Demo should be a reference for best practices. Using Foo is not a best practice 😁